### PR TITLE
HBASE-22610 [BucketCache] Rename hbase.offheapcache.minblocksize.

### PR DESCRIPTION
**Pull request by syedmurtazahassan**

### CODE REVIEW

**Comment by syedmurtazahassan** [comment]
A small comment, i am not sure if these are the required changes for this ticket. But still kindly review when anyone has time. I tried to understand what else i can do but couldnt make other changes.

**Comment by Reidddddd** [comment]
Usually, we don't actual remove an old attribute, the way we prefer is marking the old attribute deprecated, then introduce a new one, and LOG warn message and remind them of the new one when user uses the old attribute.

**Comment by syedmurtazahassan** [comment]
@Reidddddd 
Thanks for the detailed reply. I will look into this and also see if there is an already existing example and whow it is preferred.

**Comment by Reidddddd** [comment]
@syedmurtazahassan There must be some in codebase, expecting your next pull request.

**Comment by syedmurtazahassan** [comment]
@Reidddddd 
When you said we dont actually remove the old attribbute does it mean that i should not remove the old attributes which are using this attribute. 

For example here is my understanding;
BLOCKCACHE_BLOCKSIZE_KEY will be deprecated and will have a new name like BLOCKCACHE_BLOCKSIZE_KEY_NEW. 

Then further in the code BLOCKCACHE_BLOCKSIZE_KEY was used in a variable named blockSize. Now this variable will also be deprecated to something like blockSizeNew and even further in the code blockSize is used in a variable name bucketCache which will also be deprecated to bucketCacheNew. 

Is my understanding correct? All the related variables will also be deprecated right?

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 162 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 262 | master passed |
| +1 | compile | 52 | master passed |
| +1 | checkstyle | 69 | master passed |
| +1 | shadedjars | 282 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 206 | master passed |
| +1 | javadoc | 35 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 257 | the patch passed |
| +1 | compile | 55 | the patch passed |
| +1 | javac | 55 | the patch passed |
| +1 | checkstyle | 67 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 266 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 734 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | findbugs | 208 | the patch passed |
| +1 | javadoc | 33 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 14440 | hbase-server in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 17458 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 848f0a16fe82 4.4.0-137-generic #163-Ubuntu SMP Mon Sep 24 13:14:43 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / cc38de1a39 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/1/testReport/ |
| Max. process+thread count | 4933 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Reidddddd** [comment]
Please refer to [HBASE-22598](https://issues.apache.org/jira/browse/HBASE-22598), it is a typical example. Codes are better than words.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 36 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 248 | master passed |
| +1 | compile | 51 | master passed |
| +1 | checkstyle | 64 | master passed |
| +1 | shadedjars | 258 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 200 | master passed |
| +1 | javadoc | 31 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 241 | the patch passed |
| +1 | compile | 52 | the patch passed |
| +1 | javac | 52 | the patch passed |
| +1 | checkstyle | 65 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 265 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 736 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | findbugs | 215 | the patch passed |
| +1 | javadoc | 34 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 1587 | hbase-server in the patch failed. |
| +1 | asflicense | 12 | The patch does not generate ASF License warnings. |
| | | 4387 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | TEST-null |
|   | hadoop.hbase.coprocessor.TestCoprocessorInterface |
|   | hadoop.hbase.master.procedure.TestMasterProcedureScheduler |
|   | hadoop.hbase.regionserver.TestHRegionInfo |
|   | hadoop.hbase.regionserver.TestRecoveredEditsReplayAndAbort |
|   | hadoop.hbase.quotas.TestTableQuotaViolationStore |
|   | hadoop.hbase.master.snapshot.TestSnapshotManager |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 3d5671ff9baa 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 72e58a8fed |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/2/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/2/testReport/ |
| Max. process+thread count | 684 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by syedmurtazahassan** [comment]
@Reidddddd 
Added a new commit. Kindly check when you have time. Thanks.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 32 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 255 | master passed |
| +1 | compile | 53 | master passed |
| +1 | checkstyle | 67 | master passed |
| +1 | shadedjars | 270 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 208 | master passed |
| +1 | javadoc | 34 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 238 | the patch passed |
| +1 | compile | 52 | the patch passed |
| +1 | javac | 52 | the patch passed |
| -1 | checkstyle | 70 | hbase-server: The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0) |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 264 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 731 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | findbugs | 216 | the patch passed |
| +1 | javadoc | 32 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 8306 | hbase-server in the patch passed. |
| +1 | asflicense | 24 | The patch does not generate ASF License warnings. |
| | | 11161 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.8 Server=18.09.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 783b2d87b4fb 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 00075ea4fc |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/3/artifact/out/diff-checkstyle-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/3/testReport/ |
| Max. process+thread count | 4770 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Reidddddd** [comment]
LGTM，will commit tomorrow if no further comment from other members.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java [hunk 1]
```diff
@@ -17,9 +17,6 @@
  */
 package org.apache.hadoop.hbase.io.hfile;
 
-import static org.apache.hadoop.hbase.HConstants.BUCKET_CACHE_IOENGINE_KEY;
-import static org.apache.hadoop.hbase.HConstants.BUCKET_CACHE_SIZE_KEY;
```
Comment (Reidddddd) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java]: Why this two configs got deleted?

Comment (syedmurtazahassan) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java]: I accidentally included import static org.apache.hadoop.hbase.HConstants.* and removed the above two ones and didnt notice that. But now it is clean. Thanks.

Comment (syedmurtazahassan) [reply hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java]: Please find the latest commit with the changes.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 57 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 234 | master passed |
| +1 | compile | 54 | master passed |
| +1 | checkstyle | 73 | master passed |
| +1 | shadedjars | 258 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 32 | master passed |
| 0 | spotbugs | 240 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 239 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 237 | the patch passed |
| -1 | compile | 26 | hbase-server in the patch failed. |
| -1 | javac | 26 | hbase-server in the patch failed. |
| -1 | checkstyle | 70 | hbase-server: The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0) |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 259 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 717 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 31 | the patch passed |
| +1 | findbugs | 238 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 20180 | hbase-server in the patch failed. |
| +1 | asflicense | 21 | The patch does not generate ASF License warnings. |
| | | 23034 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestSnapshotTemporaryDirectory |
|   | hadoop.hbase.TestMetaTableAccessor |
|   | hadoop.hbase.client.TestConnection |
|   | hadoop.hbase.replication.TestReplicationKillSlaveRSWithSeparateOldWALs |
|   | hadoop.hbase.replication.regionserver.TestRegionReplicaReplicationEndpoint |
|   | hadoop.hbase.replication.TestReplicationSmallTestsSync |
|   | hadoop.hbase.replication.TestReplicationStatusAfterLagging |
|   | hadoop.hbase.master.procedure.TestTruncateTableProcedure |
|   | hadoop.hbase.client.TestSnapshotDFSTemporaryDirectory |
|   | hadoop.hbase.client.TestCloneSnapshotFromClientNormal |
|   | hadoop.hbase.util.TestFromClientSide3WoUnsafe |
|   | hadoop.hbase.replication.TestReplicationSmallTests |
|   | hadoop.hbase.replication.TestReplicationChangingPeerRegionservers |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.8 Server=18.09.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux b316fc8ae318 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-386/out/precommit/personality/provided.sh |
| git revision | master / c9293b0b12 |
| Default Java | 1.8.0_181 |
| compile | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/artifact/out/patch-compile-hbase-server.txt |
| javac | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/artifact/out/patch-compile-hbase-server.txt |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/testReport/ |
| Max. process+thread count | 5209 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/4/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 53 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 313 | master passed |
| +1 | compile | 71 | master passed |
| +1 | checkstyle | 89 | master passed |
| +1 | shadedjars | 354 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 42 | master passed |
| 0 | spotbugs | 323 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 322 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 325 | the patch passed |
| +1 | compile | 73 | the patch passed |
| +1 | javac | 73 | the patch passed |
| +1 | checkstyle | 106 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 384 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1116 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 41 | the patch passed |
| +1 | findbugs | 336 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 17178 | hbase-server in the patch failed. |
| +1 | asflicense | 35 | The patch does not generate ASF License warnings. |
| | | 21277 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.regionserver.TestRowTooBig |
|   | hadoop.hbase.regionserver.TestAtomicOperation |
|   | hadoop.hbase.regionserver.TestMultiColumnScannerWithNoneAndNoDataEncoding |
|   | hadoop.hbase.regionserver.TestRegionReplicasAreDistributed |
|   | hadoop.hbase.wal.TestFSHLogProvider |
|   | TEST-null |
|   | hadoop.hbase.regionserver.TestHRegionWithInMemoryFlush |
|   | hadoop.hbase.regionserver.TestHRegionFileSystem |
|   | hadoop.hbase.regionserver.wal.TestLogRollingNoCluster |
|   | hadoop.hbase.replication.regionserver.TestWALEntryStream |
|   | hadoop.hbase.regionserver.TestRegionServerAbortTimeout |
|   | hadoop.hbase.regionserver.wal.TestLogRollPeriod |
|   | hadoop.hbase.wal.TestBoundedRegionGroupingStrategy |
|   | hadoop.hbase.regionserver.throttle.TestFlushWithThroughputController |
|   | hadoop.hbase.regionserver.TestRegionFavoredNodes |
|   | hadoop.hbase.TestPartialResultsFromClientSide |
|   | hadoop.hbase.regionserver.compactions.TestCompactedHFilesDischarger |
|   | hadoop.hbase.regionserver.TestScannerHeartbeatMessages |
|   | hadoop.hbase.replication.regionserver.TestReplicator |
|   | hadoop.hbase.wal.TestSyncReplicationWALProvider |
|   | hadoop.hbase.coprocessor.TestCoprocessorStop |
|   | hadoop.hbase.regionserver.TestLogRoller |
|   | hadoop.hbase.regionserver.TestTags |
|   | hadoop.hbase.regionserver.wal.TestAsyncFSWAL |
|   | hadoop.hbase.regionserver.TestCompactionLifeCycleTracker |
|   | hadoop.hbase.wal.TestWALOpenAfterDNRollingStart |
|   | hadoop.hbase.regionserver.wal.TestAsyncLogRollPeriod |
|   | hadoop.hbase.master.procedure.TestSCPWithReplicas |
|   | hadoop.hbase.regionserver.wal.TestWALReplayBoundedLogWriterCreation |
|   | hadoop.hbase.replication.regionserver.TestSerialReplicationEndpoint |
|   | hadoop.hbase.TestClusterBootOrder |
|   | hadoop.hbase.regionserver.wal.TestSecureWALReplay |
|   | hadoop.hbase.coprocessor.TestMasterCoprocessorExceptionWithAbort |
|   | hadoop.hbase.regionserver.throttle.TestCompactionWithThroughputController |
|   | hadoop.hbase.replication.regionserver.TestReplicationSource |
|   | hadoop.hbase.regionserver.TestRegionReplicasWithRestartScenarios |
|   | hadoop.hbase.io.hfile.TestCacheOnWrite |
|   | hadoop.hbase.coprocessor.TestRegionObserverBypass |
|   | hadoop.hbase.regionserver.wal.TestLogRolling |
|   | hadoop.hbase.wal.TestWALRootDir |
|   | hadoop.hbase.regionserver.TestNewVersionBehaviorFromClientSide |
|   | hadoop.hbase.wal.TestWALSplitCompressed |
|   | hadoop.hbase.regionserver.TestDeleteMobTable |
|   | hadoop.hbase.regionserver.TestFSErrorsExposed |
|   | hadoop.hbase.regionserver.TestMobStoreScanner |
|   | hadoop.hbase.regionserver.wal.TestDurability |
|   | hadoop.hbase.TestClusterPortAssignment |
|   | hadoop.hbase.wal.TestWALSplit |
|   | hadoop.hbase.coprocessor.TestAppendTimeRange |
|   | hadoop.hbase.regionserver.TestRegionServerNoMaster |
|   | hadoop.hbase.regionserver.TestScannerRetriableFailure |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.8 Server=18.09.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 665146ecde3a 4.4.0-154-generic #181-Ubuntu SMP Tue Jun 25 05:29:03 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-386/out/precommit/personality/provided.sh |
| git revision | master / c9293b0b12 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/5/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/5/testReport/ |
| Max. process+thread count | 4691 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/5/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 32 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 242 | master passed |
| +1 | compile | 53 | master passed |
| +1 | checkstyle | 72 | master passed |
| +1 | shadedjars | 260 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 32 | master passed |
| 0 | spotbugs | 233 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 230 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 230 | the patch passed |
| +1 | compile | 52 | the patch passed |
| +1 | javac | 52 | the patch passed |
| +1 | checkstyle | 74 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 257 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 703 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 32 | the patch passed |
| +1 | findbugs | 242 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 8349 | hbase-server in the patch passed. |
| +1 | asflicense | 21 | The patch does not generate ASF License warnings. |
| | | 11178 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.8 Server=18.09.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 064117f009de 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-386/out/precommit/personality/provided.sh |
| git revision | master / aa9679ddbc |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/6/testReport/ |
| Max. process+thread count | 4833 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/6/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 64 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 271 | master passed |
| +1 | compile | 56 | master passed |
| +1 | checkstyle | 78 | master passed |
| +1 | shadedjars | 297 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 34 | master passed |
| 0 | spotbugs | 248 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 244 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 294 | the patch passed |
| +1 | compile | 58 | the patch passed |
| +1 | javac | 58 | the patch passed |
| +1 | checkstyle | 75 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 278 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 792 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 33 | the patch passed |
| +1 | findbugs | 253 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 13737 | hbase-server in the patch failed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 16933 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.0 Server=19.03.0 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/386 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 02c2786b8a6e 4.4.0-137-generic #163-Ubuntu SMP Mon Sep 24 13:14:43 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-386/out/precommit/personality/provided.sh |
| git revision | master / 9615c644f5 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/9/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/9/testReport/ |
| Max. process+thread count | 4655 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-386/9/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**